contrib/plugins/donations/LICENSE

```diff
@@ -0,0 +1,22 @@
+Note: the modules in the ccan/ directory have their own licenses, but
```

Comment:
I am uncertain why this particular note is here, when the code is in Python and does not (directly) use ccan, or have a `ccan/` subdir within this dir.

Reply:
Cause I copied the file from rusty and forgot to remove this stament. Will fix this later

Reply:
Ping